hw/mcu/dialog/cmac/pkg.yml

```diff
@@ -31,3 +31,5 @@ pkg.deps:
     - "@apache-mynewt-core/hw/hal"
     - "@apache-mynewt-core/hw/mcu/dialog"
     - "@apache-mynewt-nimble/nimble/drivers/dialog_cmac"
+
+pkg.cflags: -Irepos/apache-mynewt-core/kernel
```

Comment:
iirc `-I@apache-mynewt-core/kernel` should also work

Reply:
done